pathspider/cmd/measure.py

```diff
@@ -135,8 +147,16 @@ def _format_action(self, action):
     parser.add_argument('--output', default='/dev/stdout', metavar='OUTPUTFILE',
                         help=("The file to output results data to. "
                               "Defaults to standard output."))
+    parser.add_argument('--autoname', action='store_true',
+                        help=("Gives output file a generated name. Overwrites --output"))
     parser.add_argument('--output-flows', action='store_true',
                         help="Include flow results in output.")
+    parser.add_argument('--upload', nargs=2, metavar=('CAMPAIGN', 'API-TOKEN'),
```

Comment:
As I said above, this shouldn't be here. All of the non-MAMI users of PATHspider that I know of do not use the PTO so we should avoid having tight coupling with it in the measurement code and only aim for compatibility in the data formats.